pkg/kube/exec.go

```diff
@@ -134,9 +182,14 @@ func execStream(kubeCli kubernetes.Interface, config *restclient.Config, options
 			req.URL(),
 			config,
 			options.Stdin,
-			options.Stdout,
-			options.Stderr,
+			stdout,
+			stderr,
 			tty)
+
+		if err != nil {
```

Comment:
Is that necessary here?
It seems like this will change the behaviour for `ExecWithOptions` so it always returns error, should it be left in pod_command_executor?
Am I missing something here?